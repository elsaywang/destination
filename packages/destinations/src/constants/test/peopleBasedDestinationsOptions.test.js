import {
    peopleBasedDestinationsTypeOptions,
    getPeopleBasedDestinationsTypeLabel,
} from '../peopleBasedDestinationsOptions';

describe('peopleBasedDestinationsTypeOptions', () => {
    it('matches snapshot', () => {
        expect(peopleBasedDestinationsTypeOptions).toMatchSnapshot();
    });
    it('should return the correct label name given a valid destinaion type value', () => {
        const actual = getPeopleBasedDestinationsTypeLabel('FB');
        const expected = `FACEBOOK`;

        expect(actual).toEqual(expected);
    });
    it('should return `` given an invalid destinaion type value', () => {
        const actual = getPeopleBasedDestinationsTypeLabel('YAHOO');
        const expected = ``;

        expect(actual).toEqual(expected);
    });
});
