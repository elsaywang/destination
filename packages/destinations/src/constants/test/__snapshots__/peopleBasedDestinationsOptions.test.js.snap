// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`peopleBasedDestinationsTypeOptions matches snapshot 1`] = `
Array [
  Object {
    "label": "ALL",
    "value": "ALL",
  },
  Object {
    "label": "FACEBOOK",
    "value": "FB",
  },
  Object {
    "label": "GOOGLE",
    "value": "GOOGL",
  },
  Object {
    "label": "LINKEDIN",
    "value": "LNKD",
  },
  Object {
    "label": "TWITTER",
    "value": "TWTR",
  },
]
`;
